### PR TITLE
Update export_dashboards.go

### PR DESCRIPTION
**Pull request by hakan-carlsson**
Added support to include index-pattern in output JSON. Also added -quiet option to make it silent.

### CODE REVIEW

File: dev-tools/cmd/dashboards/export_dashboards.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func ExtractIndexPattern(body []byte) ([]byte, error) {
 		if !ok {
 			return nil, fmt.Errorf("type key not found or not string")
 		}
-		if _type != "index-pattern" {
+		if _type != "index-pattern" || indexPattern == true {
```
Comment (kvch) [inline dev-tools/cmd/dashboards/export_dashboards.go]: Why not write `|| indexPattern` instead of `|| indexPattern == true`?

Comment (hakan-carlsson) [reply dev-tools/cmd/dashboards/export_dashboards.go]: Yes, works too. IMHO I like to be explicit.

File: dev-tools/cmd/dashboards/export_dashboards.go [hunk 2]
```diff
@@ -68,7 +68,7 @@ func Export(client *http.Client, conn string, dashboard string, out string) erro
 	params.Add("dashboard", dashboard)
 
 	fullURL := makeURL(conn, exportAPI, params)
-	fmt.Printf("Calling HTTP GET %v\n", fullURL)
+	if ! quiet { fmt.Printf("Calling HTTP GET %v\n", fullURL) }
```
Comment (kvch) [inline dev-tools/cmd/dashboards/export_dashboards.go]: Could you please add new lines after `{` and `fullURL)`?

Most of the codebase does not use this format. So in order to keep the uniformity of the code, please change it.

File: dev-tools/cmd/dashboards/export_dashboards.go [hunk 3]
```diff
@@ -95,7 +95,7 @@ func Export(client *http.Client, conn string, dashboard string, out string) erro
 
 	err = ioutil.WriteFile(out, body, 0666)
 
-	fmt.Printf("The dashboard %s was exported under the %s file\n", dashboard, out)
+	if ! quiet { fmt.Printf("The dashboard %s was exported under the %s file\n", dashboard, out) }
```
Comment (kvch) [inline dev-tools/cmd/dashboards/export_dashboards.go]: Same here with the newlines.
```
if !quiet {
    fmt.Printf("The dashboard %s was exported under the %s file\n", dashboard, out)
}
```

**Comment by kvch** [comment]
jenkins test this

File: dev-tools/cmd/dashboards/export_dashboards.go (hunk 1 above)
Comment (kvch) [reply dev-tools/cmd/dashboards/export_dashboards.go]: I checked other parts of the code. Although, I find it weird, other parts of the code does include bool checking using `== true`, so I am ok with letting this go.

File: dev-tools/cmd/dashboards/export_dashboards.go (hunk 3 above)
Comment (hakan-carlsson) [reply dev-tools/cmd/dashboards/export_dashboards.go]: ok, point taken.

Comment (hakan-carlsson) [reply dev-tools/cmd/dashboards/export_dashboards.go]: made changes to indents, removed explicit =='s and removed explict bool

**Comment by kvch** [comment]
jenkins. test this, please

**Comment by kvch** [comment]
Could you please add your change to `CHANGELOG.asciidoc`?

**Comment by kvch** [comment]
Thanks.

**Pull request merged**